File: Python/1013.py
ipt = input().split()
a = int(ipt[0])
b = int(ipt[1])
c = int(ipt[2])

def maior(x,y):
    return (x+y+abs(x-y))/2

maior1 = maior(a,b)
maiorFinal = int(maior(maior1,c))
print(f'{maiorFinal} eh o maior')

File: Ruby/1_Iniciante/1013.rb
ipt = gets.split()
a = ipt[0].to_i
b = ipt[1].to_i
c = ipt[2].to_i

def maior(x,y)
    return (x+y+(x-y).abs)/2
end

maior1 = maior(a,b)
maiorFinal = maior(maior1,c).to_i
puts "#{maiorFinal} eh o maior"
